.github/workflows/nightly-2.yml

```diff
@@ -29,7 +29,7 @@ jobs:
             cd /dynawo
             export DYNAWO_HOME=$(pwd)
             util/envDynawo.sh build-user
-            sed -i 's/dtw_exceptions = {}/dtw_exceptions = {"PmConstAc7cPss2c" : 1.5, "GFM_Fault" : 1, "EntsoeTestCase3PlayBack":6.5, "HVDC":1, "PmConstAc6aPss3b":1}/' util/nrt_diff/settings.py
+            sed -i 's/dtw_exceptions = {}/dtw_exceptions = {"PmConstAc7cPss2c" : 2, "GFM_Fault" : 1, "EntsoeTestCase3PlayBack":6.5, "HVDC":1, "PmConstAc6aPss3b":1}/' util/nrt_diff/settings.py
```

Comment:
why this cahnge here?

Reply:
whoups, it's not related, this is to help with the nightly. I can put it in another PR

Reply:
yes, please :) thanks!